astropy/cosmology/tests/test_units.py

```diff
@@ -217,3 +217,164 @@ def test_equivalency_context_manager():
     # check starting with only the dimensionless_redshift equivalency.
     assert len(base_registry.equivalencies) == 1
     assert str(base_registry.equivalencies[0][0]) == "redshift"
+
+
+@pytest.mark.skipif(not HAS_SCIPY, reason="Cosmology needs scipy")
+class Test_unsuitable_units:
+#Ensure that unsuitable units are rejected
```

Comment:
```suggestion
    """Ensure that unsuitable units are rejected."""
```

Reply:
Still wrong indentation for the comment.

Reply:
Resolved.